Sources/XCTest/Public/Asynchronous/XCTWaiter.swift

```diff
@@ -369,11 +365,7 @@ private extension XCTWaiter {
 
     func cancelPrimitiveWait() {
         guard let runLoop = runLoop else { return }
-#if os(Windows)
         runLoop._stop()
```

Comment:
I notice `RunLoop._stop()` is marked `deprecated`:

https://github.com/apple/swift-corelibs-foundation/blob/main/Sources/Foundation/RunLoop.swift#L238

Does this change mean there will be a deprecation warning here at this call site in XCTest? I'm not aware of any way to "silence" such deprecation warnings, so just curious since this feels somewhat unfortunate.

Reply:
Yes, that's the only drawback, it now puts out a deprecation warning saying this function is "For XCTest use only." The good news is that it clearly says it's okay for XCTest, so nobody will be confused when they see it.

[The plan is to remove CF as a public API for all non-Darwin platforms](https://forums.swift.org/t/formalizing-the-unavailability-of-core-foundation/40216), so this is doing that. I figure that once a public API is decided, this can be changed to that. In the meantime, this avoids breakage on platforms like Android and Windows where CF APIs were just removed.

An alternative solution would be to keep it the way it is now and just extend the non-CF platforms using `RunLoop._stop()` whenever needed, while keeping Darwin and linux using the CF API for now.